frontend/packages/operator-lifecycle-manager/src/components/clusterserviceversion.tsx

```diff
@@ -234,13 +288,43 @@ export const ClusterServiceVersionTableRow = withFallback<ClusterServiceVersionT
           </Link>
         </TableData>
 
-        {/* Namespace */}
-        <TableData className={tableColumnClasses[1]}>
-          <ResourceLink kind="Namespace" title={namespace} name={namespace} />
+        {/* Operator Namespace */}
+        {ns === ALL_NAMESPACES_KEY ? (
```

Comment:
I'm tempted to say we should just always show the column until we can solve this generally for all resources. Ideally I'd like the logic to handle the namespace column in one place in table.tsx instead of having to duplicate the logic in every list view, and I don't want to hold up the important operator table changes on that. Can we address this as a follow on?